python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -32,6 +37,183 @@ class SGDBackendError(Exception):
     """Errors with BackendExecutor that should not be exposed to user."""
 
 
+class CheckpointManager:
+    """
+    Manages checkpoint processing, writing, and loading.
```

Comment:
```suggestion
    """Manages checkpoint processing, writing, and loading.
```

Reply:
Can you also document the directory structure somewhere?

Reply:
Made the changes!